recipes/yaml-cpp/all/conanfile.py

```diff
@@ -27,19 +32,20 @@ def config_options(self):
             del self.options.fPIC
 
     def configure(self):
-        if self.settings.compiler == "Visual Studio" and self.settings.compiler.version == "12":
-            raise ConanInvalidConfiguration("Visual Studio 12 not supported: Library needs C++11 standard")
+        tools.check_min_cppstd(self, "11")
```

Comment:
Check only if the value is provided. I know we need a feature in CCI to improve this situation.

```suggestion
        if self.settings.compiler.cppstd:
            tools.check_min_cppstd(self, "11")
```

Reply:
Done.